src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+function Calculator() {
+  return (
+    <section className="calculator-section">
+      <h1 className="calculator-title">Calculator App</h1>
+      <div className="calculator">
+        <div className="calculator-display"> 0 </div>
+        <div className="calculator-keys">
+          <button type="button" className="btn btn-up ">AC</button>
+          <button type="button" className="btn btn-up ">+/-</button>
+          <button type="button" className="btn btn-up ">%</button>
+          <button type="button" className="btn btn-orange ">÷</button>
+          <button type="button" className="btn">7</button>
+          <button type="button" className="btn">8</button>
+          <button type="button" className="btn">9</button>
+          <button type="button" className="btn btn-orange ">x</button>
+          <button type="button" className="btn">4</button>
+          <button type="button" className="btn">5</button>
+          <button type="button" className="btn">6</button>
+          <button type="button" className="btn btn-orange ">-</button>
+          <button type="button" className="btn">1</button>
+          <button type="button" className="btn">2</button>
+          <button type="button" className="btn">3</button>
+          <button type="button" className="btn btn-orange ">+</button>
+          <button type="button" className="btn btn-zero">0</button>
+          <button type="button" className="btn">.</button>
+          <button type="button" className="btn btn-orange ">=</button>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] Great job here 💯 But according to the requirements, You should use class based components (All components should be class-based). so kindly update this component 👍

![0](https://user-images.githubusercontent.com/45231276/204701046-f24542bc-80c5-4534-bf94-23a1002df23b.PNG)

- [ ] I really liked your design, It looks great. However, According to the requirements, The `Calculator.js` component should match the following given design ⬇

![image](https://user-images.githubusercontent.com/45231276/204702430-89bc42e6-a6a3-436b-8d1f-2261cc43bd9d.png)